exec/fixup_go119.go

```diff
@@ -0,0 +1,35 @@
+//go:build go1.19
+// +build go1.19
+
+/*
+Copyright 2022 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package exec
+
+import (
+	osexec "os/exec"
+	"strings"
+)
+
+// fixupGo119Behavior reverts the behavior of osexec.Cmd to what it was before go1.19
+// specifically set the Err field to nil (LookPath returns a new error when the file
+// is resolved to the current directory.
+func fixupGo119Behavior(cmd *osexec.Cmd) *osexec.Cmd {
+	if cmd.Err != nil && strings.Contains(cmd.Err.Error(), osexec.ErrDot.Error()) {
```

Comment:
Could we use `errors.Is`?
```suggestion
	if cmd.Err != nil && errors.Is(cmd.Err, osexec.ErrDot) {
```

Reply:
yep will update in the next rev

Reply:
Done!